app/assets/javascripts/jquery_overrides.js

```diff
@@ -37,10 +37,20 @@ $.ajaxSetup({
     }),
     "text script": logError(function (text) {
       if (text.match(/^{/)) {
+        console.debug('json payload');
         return jQuery.jsonPayload(text, function (text) {
           return text;
         });
+
+      } else if (text.match(/<!DOCTYPE html>/)) { // plain HTML payload fallback
```

Comment:
Problematic .. maybe `/^<!DOCTYPE html>/`

(because any JS that contains that string would match too..)